cmd/fs-v1-metadata.go

```diff
@@ -315,37 +317,63 @@ func createFormatFS(fsPath string) error {
 }
 
 func initFormatFS(fsPath string) (rlk *lock.RLockedFile, err error) {
+	// Done channel is used to close any lingering retry routine, as soon
```

Comment:
Basically the conditions which don't reach in codecoverage right now need to have a situation where locks are held by a separate process so that the fcntl locks can be honored. Holdings fcntl locks on the same process have no meaning @balamurugana 

So it can't be added..